docs/ig.md

```diff
@@ -189,12 +189,6 @@ suggests to implement Debugging Profiles (`--profile=`) to be able to give the n
 kubectl v1.27 implements some of those profiles but not yet the "sysadmin" profile, so it is not possible to use
 `--profile=` yet.
```

Comment:
This paragraph is out of date. It can be deleted. Instead, you could just say something like:
> This requires Kubernetes v1.30.0. It is also working on older versions as long as kubectl (client) is v1.30.0 or greater.